front/src/common/RollingStockSelector/RollingStockCardButtons.js

```diff
@@ -40,6 +42,10 @@ export default function RollingStockCardButtons(props) {
     return options;
   };
 
+  useEffect(() => {
```

Comment:
Please add the rollingStockConmfort dependency in the array

Reply:
done